kolibri/plugins/audio_mp3_render/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "kolibri-audio-mp3-render-plugin",
+  "description": "Audio/MP3 Render Plugin for Kolibri",
+  "private": true,
+  "dependencies": {
+    "html5media": "1.1.8"
+  }
+}
```

Comment:
This is really great -- should we open an issue about versioning Kolibri as well? Not as an NPM dependency, but more a separate thing that stuff breaks when it's not built with the right Kolibri?